packages/suite-build/webpack.config.ts

```diff
@@ -8,6 +8,7 @@ import base from './configs/base.webpack.config';
 import dev from './configs/dev.webpack.config';
 import web from './configs/web.webpack.config';
 import desktop from './configs/desktop.webpack.config';
+import landing from './configs/landing.webpack.config';
 
 const configs = [base];
```

Comment:
Beware, that base config uses `browserslist` with
```
Chrome >= 59
Firefox >= 54
```
Landing page should be supported in most browsers. https://browserslist.dev/